<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatPanel</class>
 <widget class="QWidget" name="ChatPanel">
  <layout class="QVBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="headerFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>56</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>56</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="headerLayout">
      <property name="leftMargin">
       <number>16</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>16</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>微信</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="headerSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="emptyPage">
      <layout class="QVBoxLayout" name="emptyLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="emptyTopSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>80</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="emptyIconLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="emptyHintLabel">
         <property name="text">
          <string>未选择聊天</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="emptyBottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>120</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="chatPage">
      <layout class="QVBoxLayout" name="chatPageLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="chatView">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="inputFrame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>58</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="inputLayout">
          <property name="leftMargin">
           <number>14</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>14</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QLineEdit" name="messageEdit">
            <property name="placeholderText">
             <string>输入消息，按回车发送...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
